modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -920,6 +920,31 @@ CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-2.0'}
 ----
 ====
 
+.Using differently ordered return items in a `UNION` clause
+====
+Query::
++
+[source,cypher]
+----
+RETURN 'val' as one, 'val' as two
+UNION
+RETURN 'val' as two, 'val' as one
+----
+
+Description of the returned code::
+All sub queries in a UNION should have the same ordering for the return column names, this is deprecated and will be enforced in later versions.
```

Comment:
Also do we want to mention that this also applies to `UNION ALL` in the description?

Reply:
Right, so in the deprecation docs I added examples for UNION ALL as well, but wasn't sure if it was overkill to add it here as well, but maybe

All subqueries in a UNION [ALL] should have... 

would work perhaps?

Reply:
Yes, I think you don't need to add another example, I like writing it like `UNION [ALL]` would be nice